src/reducers/inputs.js

```diff
@@ -42,6 +42,31 @@ export const setMyItems = (state, action) => {
   return newState
 }
 
+export const setMyItemsLang = (state, action) => {
+  const newState = { ...state }
+  const reduxPath = action.payload.reduxPath
+  let level = 0
+
+  reduxPath.reduce((obj, key) => {
+    level++
+    if (level === reduxPath.length) {
+      if ((key in obj) != true) {
+        obj[key] = { items: [] }
+      }
+      obj[key].items.forEach(item => {
+        if (item.id == action.payload.id) {
```

Comment:
Is this only going to match one item in the array or multiple? (Not sure what this data tends to look like.) If only the first, we may want a different iterator function above.

Reply:
Yes, only one. Open to suggestions.

Reply:
OK, cool. How about:

```javascript
const payloadItem = obj[key].	items.find(item => item.id === action.payload.id)
payloadItem.lang = { items: action.payload.items }
```

Reply:
Thanks for the suggestion. Updated.